src/OrleansProviders/OrleansProviders.csproj

```diff
@@ -100,7 +100,7 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="SQLServer\CreateTables.sql">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Any particular reason why this needs to be Copy Always?

Using PreserveNewest rather than Always gives much faster incremental build times overall.